sdk/websites/Azure.ResourceManager.AppService/tests/TestsCase/CertificatesCollectionTests.cs

```diff
@@ -39,6 +39,7 @@ public async Task CreateOrUpdate()
 
         [TestCase]
         [RecordedTest]
+        [Ignore("KeyVaultId is allowed to be null, and this test case will be adjusted according to the situation in the future")]
```

Comment:
Please don't ignore test cases that are not valid.   Test cases should be fixed/removed/added along with the feature that they're testing.